benchmark/Dockerfile

```diff
@@ -3,8 +3,11 @@ FROM ubuntu:22.04
 ENV CSV_DIR /csv
 ENV SERIALIZED_DIR /serialized
 
+RUN wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
```

Comment:
The Dockerfile does not seem to be reverted correctly. @ray6080 could you please revert this before merging the PR?

Reply:
Yeah. nice catch. thanks!
I changed it now.